src/NHSD.BuyingCatalogue.Organisations.Api/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using HealthChecks.Network.Core;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using NHSD.BuyingCatalogue.Identity.Common.Constants;
+using NHSD.BuyingCatalogue.Organisations.Api.Settings;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Extensions
+{
+    internal static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection RegisterHealthChecks(this IServiceCollection services, string connectionString, SmtpSettings smtpSettings)
```

Comment:
Can `connectionString` or `smtpSettings` parameters be null? I think this one will come up in the coding standards meeting tomorrow but we made @bjss-chris change his code to guard & fail fast with parameters that should not be null.